docs/sso_via_saml_mscolab.rst

```diff
@@ -0,0 +1,552 @@
+SSO via SAML Integration Guide for MSColab Server
+=================================================
+
+In this documentation, you will go through the following topics.
+
+    1. Introduction
+    
+    2. Configuring an existing IdP
+    
+        * Private key and certificate
+        
+        * Configuring MSColab settings
+        
+            * MSColab configurations
+            * Establish pysaml2, Saml2Client for the MSColab server
+            
+        * Configuration `mss_saml2_backend.yaml` file
+        
+        * Access SAML2Client metadata of MSColab
+        
+        * Guide to IDP Configuration
+
+    3. Configuration example through Keycloak 13.0.1
+    
+        * Setting Up Keycloak
+        
+            * Installation and run Keycloak
+            * Setup Keycloak IdP
+            
+        * Configure MSColab server
+        
+            * Configuration in MSColab settings for Keycloak
+            * Configuration `mss_saml2_backend.yaml` file
+
+    4. Configuration Multiple IDPs
+
+1. Introduction
+***************
+This documentation will explain how to configure MSColab with an existing IdP or multiple IdPs, along with examples of implementation.
+
+If you are not aware of how the SAML process works in the MSColab server, it is highly recommended to set up msidp and test it with MSColab as an initial step before configuring existing 3rd party IdPs.
+
+.. note::
+    You can find instructions to set up msidp by `conf_sso_test_msscolab.rst`.
+
+
+2. Configuring an existing IdP
+******************************
+
+To configure an existing IdP, you will need a signed certificate and a private key for the MSColab server. Additionally, you will require metadata for the IdP to complete the configuration.
+
+Furthermore, you will need to configure saml2 setup in your `setup_saml2_backend.py` file and configure settings in your `mscolab_settings.py` file. On development you need only to use your PYTHONPATH and `setup_saml2_backend.py`, `mscolab_settings.py` in that path.
+
+.. note::
+    When you want to set a parameter or change a default add it to that file,
+    
+    eg:-
+
+        $ more mscolab_settings.py
+        
+        USE_SAML2 = True
```

Comment:
only when this is set to True, the saml2 urls should be accessible, please change this in a different PR

Reply:
done.. #2068